be/src/exprs/agg/window.h

```diff
@@ -233,6 +233,44 @@ class DenseRankWindowFunction final : public WindowFunction<DenseRankState> {
     std::string get_name() const override { return "dense_rank"; }
 };
 
+struct CumeDistState {
+    int64_t rank;
+    int64_t frame_start;
+    int64_t count;
+};
+
+class CumeDistWindowFunction final : public WindowFunction<CumeDistState> {
```

Comment:
it seems that cume_dist show should only support range-typed frame, so we must fobid rows-typed frame for cume_dist

Reply:
In this PR, the use of Windowing clause is not allowed with the `CUME_DIST()` window function, just like `Rank()` and `DENSE_Rank()`. I've forbidden it at the frontend.

```java
if (analyticExpr.getWindow() != null) {
    if ((isRankingFn(analyticFunction.getFn()) || isCumeFn(analyticFunction.getFn()) || 
            isOffsetFn(analyticFunction.getFn()) || isHllAggFn(analyticFunction.getFn()))) {
        throw new SemanticException("Windowing clause not allowed with '" + analyticFunction.toSql() + "'",
                analyticExpr.getPos());
    }

    verifyWindowFrame(analyticExpr);
}
```